### PR TITLE
Feature/cazb 2556 payment history by vrn

### PR DESCRIPTION
**Pull request by lukaszkuczynski**
New endpoint `/v1/payments/vehicles-history/{vrn}`
It is returning values from table `t_clean_air_zone_entrant_payment_match`

https://eaflood.atlassian.net/browse/CAZB-2556

### CODE REVIEW

File: src/it/java/uk/gov/caz/psr/VehiclePaymentHistoryControllerTestIT.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    givenRequestForVrn(
+        "ND84VSY")
+        .whenRequestForHistoryIsMade()
+        .pageContainsHistoryItems(1);
+  }
+
+  private VehiclePaymentHistoryJourneyAssertion givenRequestForVrn(String vrn) {
+    return new VehiclePaymentHistoryJourneyAssertion().forVrn(vrn);
+  }
+
```
Comment (jaroslawpiotrowski) [inline src/it/java/uk/gov/caz/psr/VehiclePaymentHistoryControllerTestIT.java:141]: Can you remove empty lines?

File: src/it/resources/data/sql/add-caz-entrant-payments-unsorted.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	clean_air_zone_entrant_payment_id, vrn, clean_air_zone_id, travel_date, tariff_code, charge, payment_status, update_actor)
+	VALUES
+	  ('43ea77cc-93cb-4df3-b731-5244c0de9cc8', 'ND84VSX', 'b8e53786-c5ca-426a-a701-b14ee74857d4', '2019-11-01', 'tariff_code1', 50, 'PAID', 'USER'),
+	  ('688f8278-2f0f-4710-bb7c-6b0cca04c1bc', 'ND84VSX', 'b8e53786-c5ca-426a-a701-b14ee74857d4', '2019-11-03', 'tariff_code2', 50, 'PAID', 'USER'),
+	  ('9cc2dd1a-905e-4eaf-af85-0b14f95aab89', 'ND84VSX', 'b8e53786-c5ca-426a-a701-b14ee74857d4', '2019-11-02', 'tariff_code3', 100, 'PAID', 'USER'),
+	  ('21b7049d-b978-482f-a882-4de6bb9d699c', 'ND84VSX', 'b8e53786-c5ca-426a-a701-b14ee74857d4', '2019-11-04', 'tariff_code4', 100, 'PAID', 'USER'),
+	  ('21b7049d-b978-482f-a882-4de6bb9d699d', 'ND84VSX', '53e03a28-0627-11ea-9511-ffaaee87e375', '2019-11-04', 'tariff_code4', 100, 'PAID', 'USER'),
+    ('21b7049d-b978-482f-a882-4de6bb9d699e', 'ND84VSY', '53e03a28-0627-11ea-9511-ffaaee87e370', '2019-11-04', 'tariff_code4', 100, 'PAID', 'USER')
+
+	  ;
```
Comment (jaroslawpiotrowski) [inline src/it/resources/data/sql/add-caz-entrant-payments-unsorted.sql:19]: Can you move `;` behind the bracket?

Comment (lukaszkuczynski) [reply src/it/resources/data/sql/add-caz-entrant-payments-unsorted.sql]: sure

File: src/it/java/uk/gov/caz/psr/VehiclePaymentHistoryControllerTestIT.java (hunk 1 above)
Comment (lukaszkuczynski) [reply src/it/java/uk/gov/caz/psr/VehiclePaymentHistoryControllerTestIT.java:166]: yup!

**Pull request merged**